Automattic/wp-calypso
# Stats: Replacing followers URL with subscribers

Related to #73611

## Proposed Changes

* replace links using `followers` with `subscribers` because previous `followers` and `email-followers` are not accessible anymore

## Testing Instructions



* open the live branch and to the `Insights` page
* click `view all` at the bottom of `Subscribers` module and verify that the page you are redirected to has uses visible below the header
* repeat the check when `Email` is selected in the module's title

## Pre-merge Checklist
